jdi-light-vuetify/src/main/java/com/epam/jdi/light/vuetify/elements/complex/RangeSlider.java

```diff
@@ -77,6 +82,17 @@ public List<Integer> getValue() {
                 .collect(Collectors.toList());
     }
 
+    @JDIAction("Get value from '{name}'")
+    public Integer getLeftValue() {
+        return Integer.parseInt(core().findElement(By.cssSelector("[id*='input-min']"))
+                .getAttribute("value"));
+    }
+
+    @JDIAction("Get value from '{name}'")
+    public Integer getRightValue() {
+        return Integer.parseInt(core().findElement(By.cssSelector("[id*='input-max']")).getAttribute("value"));
+    }
+
     @JDIAction("Get thumb label value from '{name}'")
     public List<String> getThumbLabelValue() {
         return getThumbLabel()
```

Comment:
Wouldn't it be better if the method getThumbLabel() will returns the Label element (elements)? 
And then we will work with this element inside the label class, and not inside the rangeslider class.